run_tests.py

```diff
@@ -50,6 +53,12 @@ def usage():
 be resumed manually from any given test.
 """)
 
+DISABLED_TESTS_FILE_NAME = "/tests_disabled.json"
+disfile = os.path.dirname(__file__) + DISABLED_TESTS_FILE_NAME
+
+disabled_reader = shell.DisabledListLoader(disfile)
+disabled_reader.try_load()
```

Comment:
Please add command line option, like `--force-disabled`. Don't make users to manually remove the file if disabled tests are required.

Reply:
disabled file have option `'disable' : 'True'`